pkg/provider-local/controller/backupoptions/options.go

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2021 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package backupoptions
+
+import (
+	"github.com/spf13/pflag"
+	"sigs.k8s.io/controller-runtime/pkg/controller"
+)
+
+const (
+	// DefaultBackupDir is the default path to the directory where the backup bucket is created.
+	DefaultBackupPath = "dev/local-backupbuckets"
+	// DefaultContainerMountDir is the default path to the directory where the backup bucket is mounted on the container.
+	DefaultContainerMountPath = "/etc/gardener/local-backupbuckets"
+)
+
+// ControllerOptions are command line options that can be set for controller.Options.
+type ControllerOptions struct {
+	// BackupBucketPath is the path to the backupbucket.
+	BackupBucketPath string
+	// ContainerMountPath is the path to the directory where the backup bucket is mounted on the container.
+	ContainerMountPath string
+
+	config *ControllerConfig
+}
+
+// AddOptions are options to apply when adding the backupbucket controller to the manager.
+type AddOptions struct {
+	// BackupBucketPath is the path to the backupbucket.
+	BackupBucketPath string
+	// ContainerMountPath is the path to the directory where the backup bucket is mounted on the container.
+	ContainerMountPath string
+	// Controller are the controller.Options.
+	Controller controller.Options
+	// IgnoreOperationAnnotation specifies whether to ignore the operation annotation or not.
+	IgnoreOperationAnnotation bool
+}
+
+// AddFlags implements Flagger.AddFlags.
+func (c *ControllerOptions) AddFlags(fs *pflag.FlagSet) {
+	fs.StringVar(&c.BackupBucketPath, "local-dir", c.BackupBucketPath, "Path to the directory where the bucket will be created.")
```

Comment:
```suggestion
	fs.StringVar(&c.BackupBucketPath, "backup-bucket-path", c.BackupBucketPath, "Path to the directory on the local host where the backup buckets will be created.")
```

Reply:
with `controllercmd.PrefixOption("backupbucket-", localBackupBucketOptions),` the the `backupbuket` prefix is already added to the property and with the proposed change it will be `--backupbuket-backup-bucket-path`. Probably we can leave it to be just `path` there?

Reply:
local-dir is ok, I saw the prefix after I wrote this comment, but then forgot to remove it, sorry.